omnivore-app/omnivore
# [Omn-159] - Floating Action Buttons - Phase 1

## Spec Checklist
- [x] Does the figma design fit the ticket description ?
- [x] Added video describing your plan of action ?

## Description

Omnivore has a design for a [floating action button](https://material.io/components/buttons-floating-action-button) for actions on an article.

This PR implements "Phase One".
"Phase One" includes:
- Adjust font size button.

## House Keeping
- [x] Added video describing your plan of action ?
- [x] Added Loom video ?
- [x] Is Deployment Link passing ?
- [x] Have you assinged PR to yourself ?
- [x] Is Github action passing ?
- [x] Updated the appropriate tag ?


## Ticket link (if applicable)

[Task Link](https://app.gitstart.com/clients/omnivore/tickets/OMN-159)


## How has this been tested?

- Sign in.
- Create an article if it doesn't exist yet.
- Open the article.
- In the bottom right there should be a "..." icon (Icon with three dots). Press it.
- Pressing that button will show more buttons, The Buttons UI, and functionality are parts of the task.
- Press the "Aa" Icon, it will show a slider. Increasing/Decreasing the slider value changes the font size of the article. (Min 10, Max 28.

## Types of changes


## Checklist:

- [ ] I have test my code on different browsers and devices to make sure it is cross-browser compatible (both desktop and mobile).
- [ ] I have confirmed the payload for integrated API matches params on API docs
- [x] I have confirmed design matches Spec specified on Figma.

## Remarks

- [x] I added a new package to achieve this task.
- [x] I have updated package.json

## Loom Video
https://www.loom.com/share/39c233697cb3467db10602ff3d40e18b
